incl/mods/requestUserAccess.php

```diff
@@ -13,6 +13,6 @@
 	if ($permState >= 2){ // if the mod badge level is higher than 2, it will still show elder mod message
 		exit("2");
 	}
-	echo $permState; 
+	echo htmlspecialchars($permState, ENT_QUOTES, 'UTF-8');
```

Comment:
most of these `htmlspecialchars` calls feel pointless, as the value is sanitized elsewhere